parsers/go.c

```diff
@@ -531,19 +547,60 @@ static int parsePackage (tokenInfo *const token)
 		return CORK_NIL;
 }
 
+static tokenInfo * parseReceiver (tokenInfo *const token)
```

Comment:
I think this function can be simplified a bit and made more readable. I'll prepare a patch - it's easier than writing what I think should be changed.

Reply:
The patch is here

https://github.com/techee/ctags/commits/scope_improvement

Feel free to squash it with your commit if you like it.